connector_flow/task/abstract_task.py

```diff
@@ -96,7 +96,10 @@ def run(self, chunk_id=None, **kwargs):
         except:
             raise
         finally:
-            chunk.write({'state': new_state})
+            try:
+                chunk.write({'state': new_state})
+            except:
+                pass
```

Comment:
I think this is not correct. If an error occurs during writing the state to the chunk I think the whole transaction will roll back, so it's missleading to silently catch the exeption here.